### PR TITLE
Fix tests to work with 16.8 localization

### PR DESCRIPTION
**Pull request by JoeRobich**
See changes in https://github.com/dotnet/roslyn/commit/41036e261fc5d2b417a5a58b176420b949ed2bb9 needed to work around differing punctuation translated strings.

/CC: @wli3

### CODE REVIEW

**Review by ryzngard — APPROVED** [review]
Approve commit: 41036e261fc5d2b417a5a58b176420b949ed2bb9

**Comment by Cosifne** [comment]
QB has approved this for P6 so let's wait until next week to merge it.

**Comment by wli3** [comment]
@Cosifne @JoeRobich note you still need time for it to flow to SDK and VS

**Pull request merged**
